test/e2e/0xCallee.ts

```diff
@@ -0,0 +1,131 @@
+// mainnet fork
+import BigNumber from 'bignumber.js'
+import {Trade0xInstance, MockERC20Instance, WETH9Instance} from '../../build/types/truffle-types'
```

Comment:
can we also test this using a ganache fork using mainnet 0x data?

Reply:
This is using the mainnet fork, you can review it first, I'm figuring it out how to set it up in the CI process.